index.js

```diff
@@ -1,70 +1,19 @@
-const listElm = document.querySelector('.books-list');
-const form = document.querySelector('.form');
-
-class Books {
-  constructor() {
-    this.list = localStorage.getItem('booksList')
-      ? JSON.parse(localStorage.getItem('booksList'))
-      : [];
-  }
-
-  addBook(book) {
-    this.list.push(book);
-
-    localStorage.setItem('booksList', JSON.stringify(this.list));
-  }
-
-  removeBook(title) {
-    this.list = this.list.filter((currentBook) => currentBook.title !== title);
-    localStorage.setItem('booksList', JSON.stringify(this.list));
-  }
-}
-
-const booksList = new Books();
-
-function renderBooks() {
-  listElm.innerHTML = '';
-
-  booksList.list.forEach((book) => {
-    listElm.innerHTML += `
-      <li>
-          <p><span class="title">${book.title}</span> by ${book.author}</p>
-          <button class="remBtn">Remove</button>        
-      </li>
-    `;
-  });
-
-  const removeBtns = document.querySelectorAll('li button');
-  removeBtns.forEach((btn) => {
-    btn.addEventListener('click', () => {
-      const elem = btn.parentNode;
-      const bookTitle = elem.querySelector('.title').textContent;
-
-      booksList.removeBook(bookTitle);
-      elem.remove();
-    });
-  });
-}
-
+/* eslint-disable import/extensions, no-unused-vars  */
+import selector from './modules/selectors.js';
+import navigator from './modules/spa-manager.js';
+import Books from './modules/bookclass.js';
```

Comment:
- [ ] I cannot navigate between pages; I'm not able to add books because the navbar doesn't work. Kindly fix this issue.
- [ ] Also, the date is not displayed below the navigation bar. Kindly work on that.

<img width="959" alt="image" src="https://user-images.githubusercontent.com/77190203/159268210-e9ffc0e7-4aa1-46df-8579-8cddeba1bc88.png">